tests/test_amity.py

```diff
@@ -0,0 +1,62 @@
+import unittest
+from amity import Amity
+
+
+class AmityTest(unittest.TestCase):
+
+    def setUp(self):
+        self.building = Amity()
+
+    def test_populate_offices(self):
+        '''
+        Test whether the populate method
+        populates the building with office rooms
+        '''
+        self.building.populate()
+        self.assertTrue(len(self.building.office_rooms) > 0)
+
+    def test_populate_living(self):
+        '''
+        Test whether the populate method
+        populates the building with living rooms
+        '''
+        self.building.populate()
+        self.assertTrue(len(self.building.living_rooms) > 0)
```

Comment:
**Code Quality: Nitpick**

Test cases should be more exact as opposed to asserting greater than 0 e.g in this case `assertEqual(len(self.building.living_rooms), 5)`.